parnold-x/nasc
# Port to Meson

## CMake: Reorganize files

1. Rename `src/config.vala.cmake` to `src/config.vala.in`

2. Move `com.github.parnold-x.nasc.gschema.xml` to `data` subdirectory

## Port to Meson

Based on #98

Fixes:
1. Fix dependencies for `libqalculatenasc`
https://github.com/parnold-x/nasc/pull/98#issuecomment-438664227

2. Remove header files from `meson.build`
https://github.com/parnold-x/nasc/pull/98#discussion_r234801513

3. Get rid of `get_option('prefix')` from `install_dir`
https://github.com/parnold-x/nasc/pull/98#discussion_r239734092

4. Make Meson optional
Do not remove CMake

---

This PR is related to submitting my flatpak package to the Flathub repository.
https://github.com/parnold-x/nasc/issues/105
https://github.com/flathub/flathub/pull/867

See also:
https://github.com/scx/nasc/pull/4
